src/styles/selectionbar.styl

```diff
@@ -0,0 +1,65 @@
+@require 'cozy-ui'
+
+slate-grey = #5d6165
+
+.fil-selectionbar
+  position fixed
+  display flex
+  justify-content center
+  align-items center
+  width 100%
+  top 0
+  left 0
+  color white
+  background-color slate-grey
+  height em(56px)
+  font-weight bold
+  z-index 1000 // because... you know...
+
+  .fil-selectionbar-separator
+    background-color black
+    border-radius 50%
+    width 4px
+    height 4px
+    margin 0 0.5em 0 2em
+
+  button
```

Comment:
Yes, the selection bar should become a cozy-ui thing, but I think it's better to wait the second time we need it, so that we can choose how to componentize it.